src/user/user.controller.ts

```diff
@@ -82,4 +83,14 @@ export class UserController {
 
     return this.repoService.findAll(pageOptionsDto, user.id, ["TopRepos"]);
   }
+
+  @Get("/top")
+  @ApiOperation({
+    operationId: "getTop10Highlights",
+    summary: "List top users",
+  })
+  @ApiOkResponse({ type: DbTopUser })
+  async getTop10Highlights(@Query() limit: number): Promise<DbTopUser[]> {
```

Comment:
Let's remove this for now

```suggestion
  async getTop10Highlights(): Promise<DbTopUser[]> {
```